yeat/cli.py

```diff
@@ -78,8 +105,11 @@ def main(args=None):
     if args is None:
         args = get_parser().parse_args()
     assert len(args.reads) == 2
+    assembly = list(filter(None, args.assembly.strip().split(",")))
```

Comment:
Takes the string passed in with `--assembly` and splits the substrings by commas. The `filter()` removes any empty strings from the list of strings.

Reply:
Clever! You may consider renaming this variable from `assembly` to `assemblers` for sake of clarity. A name like "assembly" could have any number of meanings, but "assemblers" is pretty clearly a list of one or more assembly tools/programs/algorithms.